src/gwt/src/org/rstudio/studio/client/common/spelling/ui/SpellingCustomDictionariesWidget.java

```diff
@@ -120,10 +119,11 @@ public void execute(FileSystemItem input,
                      return;
                   
                   progressIndicator_.onProgress("Adding dictionary...");
-                  
-                  spellingService_.addCustomDictionary(
-                                                  input.getPath(), 
-                                                  customDictRequestCallback_);
+
+                  // TODO: add custom dictionary
+//                  spellingService_.addCustomDictionary(
```

Comment:
I think this might be why you're just seeing a spinner when attempting to add dictionaries. ;-) Why does this need to be disabled? If you can, disable it only when real-time spell checking is enabled (use progressIndicator.onError to explain).

Reply:
Yeah oof. I didn't see this because I was diffing my changes against the committed branch and not v1.3 so I didn't see this. I had temporarily commented out all references to spellingService because I was removing it entirely in the conversion.